libraries/AP_OSD/AP_OSD.h

```diff
@@ -48,8 +49,18 @@ class AP_MSP;
 #define PARAM_INDEX(key, idx, group) (uint32_t(uint32_t(key) << 23 | uint32_t(idx) << 18 | uint32_t(group)))
 #define PARAM_TOKEN_INDEX(token) PARAM_INDEX(AP_Param::get_persistent_key(token.key), token.idx, token.group_element)
 
-#define AP_OSD_NUM_SYMBOLS 91
+#define AP_OSD_NUM_SYMBOLS 107
 #define OSD_MAX_INSTANCES 2
+
+#if AP_OSD_CRSF_EXTENSIONS_ENABLED && AP_RCPROTOCOL_CRSF_ENABLED
```

Comment:
```suggestion
#if AP_OSD_CRSF_EXTENSIONS_ENABLED
```

Far better to get a compilation failure if someone forgets the define than to silently fail.

The custom build server will take care of adding the extra define based on the dependencies you've specified in `build_options.py`

Reply:
But what if someone doesn't use the custom build server, for example building locally from a git repo clone? Wouldn't it be better to leave both conditions there to ensure the correct dependencies are set?

Reply:
Since I have renamed the main define, I took the opportunity and removed the second condition, as you suggested.